doc/reduce-memory.md

```diff
@@ -18,7 +18,7 @@ The size of some in-memory caches can be reduced. As caches trade off memory usa
 
 - Since `0.14.0`, unused memory allocated to the mempool (default: 300MB) is shared with the UTXO cache, so when trying to reduce memory usage you should limit the mempool, with the `-maxmempool` command line argument.
 
-- To disable most of the mempool functionality there is the `-blocksonly` option. This will reduce the default memory usage to 5MB and make the client opt out of receiving (and thus relaying) transactions, except from whitelisted peers and as part of blocks.
+- To disable most of the mempool functionality there is the `-blocksonly` option. This will reduce the default memory usage to 5MB and make the client opt out of receiving (and thus relaying) transactions, except from peers who have the `relay` permission set (e.g. whitelisted peers), and as part of blocks.
```

Comment:
This helps explain the motivation for developers, but for a user, there is no cause and effect relationship between what transactions might be relayed and what the default maximum capacity is.

Reply:
I think there is a clear relationship between fewer transactions to relay and less memory needed, but can drop the second sentence if that seems unclear.  The main suggestion is to begin with the memory usage, as it is the topic of the document, rather than the "disable mempool functionality" phrase (which seems a bit misleading as well).